tests/envs/task.yaml

```diff
@@ -13,3 +13,6 @@ execute: |
 
 debug: |
     cat task.out || true
+
+restore: |
+    rm -f .spread-reuse.yaml /run/spread-reuse.yaml
```

Comment:
That doesn't sound right.  What's the point of using --reuse if it's blowing the data that allows reuse away?   Why is --reuse there in the first place?

Reply:
Agree, I need just to delete .spread-reuse.yaml to be able to run with -repeat option.